File: limesz_client/src/app/api/models/resource-description.ts
/* tslint:disable */
/* eslint-disable */
import { ResourceDescriptionOptions } from './resource-description-options';
import { ResourceProp } from './resource-prop';
export interface ResourceDescription {
  descriptionOptions?: ResourceDescriptionOptions;
  name?: null | string;
  props?: null | Array<ResourceProp>;
  type?: null | string;
}

File: limesz_app/limesz_data/Models/AutoUI/ResourceDescription/ResourceDescription.cs
namespace margarita_data.Models.AutoUI;

public class ResourceDescription
{
    public string Name { get; set; }
    public string Type { get; set; }
    public List<ResourceProp> Props { get; set; }
    public ResourceDescriptionOptions DescriptionOptions { get; set; } = new ResourceDescriptionOptions();
}